### ISSUE TITLE
Idea's for smaller changes: Move external-dns to it's own module

### ISSUE THREAD

**Issue by qvistgaard** [nuuday/terraform-aws-eks#4]
Currently we run more the 1 instance in our clusters, and it will make it helpful if it is module on it's own

**Comment by wolffberg** [comment]
As the external-dns pod is almost just a Helm Chart, does it make sense to split it (and possibly the other add-ons) up into their own modules instead of disabling them and adding them to your project manually?

An alternative could be to add the output needed to add your own Helm Charts easily

```
host                   = data.aws_eks_cluster.cluster.endpoint
cluster_ca_certificate = base64decode(data.aws_eks_cluster.cluster.certificate_authority.0.data)
token                  = data.aws_eks_cluster_auth.cluster.token
load_config_file       = false
```

**Comment by qvistgaard** [comment]
I think they should be split into modules. however i still think they should standard in the eks setup.

**Comment by anarsen** [comment]
I'm not sure I know what this means
>Currently we run more the 1 instance in our clusters, and it will make it helpful if it is module on it's own

How would putting this into a module help? It's not used repeatedly within the EKS module.

I could see the benefit from focusing more on addon modules though instead of wrapping the official `terraform-aws-eks` module 1:1 like it's shaping up to be in PR #7

**Comment by anarsen** [comment]
We solved this in #9. Can you close this issue if you agree @qvistgaard?
